docs/development.md

```diff
@@ -217,6 +217,13 @@ Follow the [cleanup instructions in the user guide](userguide.md#cleanup).
 In order to test your changes on your kubernetes cluster, you'll need
 to build an image and a deployment config.
 
+**NOTE:** When federation CRDs are changed, you need to run:
+```bash
+./scripts/sync-up-helm-chart.sh
+```
+This script ensures that CI will pass.
```

Comment:
s/This script ensures that CI will pass./This script ensures that the CRD resources in helm chart can be synced.